test/test_jit.py

```diff
@@ -6486,6 +6486,25 @@ def foo(x):
 
         self.assertEqual(foo(input), input)
 
+    def test_export_dynamic_slice(self):
```

Comment:
What's dynamic about this slice? That it doesn't have statically available ends?

Reply:
Yeah, previously `starts`, `ends` and `axes` were required to be attributes in ONNX, this is dynamic in the sense that they can differ at runtime